xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/misc.less

```diff
@@ -324,6 +324,34 @@
   }
 }
 
+.changes-version {
+  white-space: nowrap;
+}
+
+.changes-arrow, .changes-arrow-big, .changes-arrow-left, .changes-arrow-right {
+  &, &:hover, &:active, &:visited, &:focus {
+    color: @xwiki-page-content-bg;
+    text-decoration: none;
+    font-weight: 700;
+  }
+}
+
+.changes-arrow-left, .changes-arrow-right {
+  font-size: 7em;
+}
+
+.changes-arrow-left {
+  position: absolute;
+  top: 0;
+  left: 30px;
+}
+
+.changes-arrow-right {
+  position: absolute;
+  top: 0;
```

Comment:
Ideally, you could have used percentages to calculate the middle of the container.

Reply:
Done, using `transform: translateY(-50%);`